python/kserve/requirements.txt

```diff
@@ -21,3 +21,4 @@ ray[serve]==2.0.0
 grpcio>=1.34.0
 tritonclient==2.18.0
 protobuf~=3.19.0
+prometheus-client>=0.13.1
```

Comment:
0.14 is the latest version, but ray[serve] requires <0.14, and this is the latest version that is <0.14